Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/interoperability/InteroperabilityConfigurationFragment.kt

```diff
@@ -0,0 +1,47 @@
+package de.rki.coronawarnapp.ui.interoperability
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import de.rki.coronawarnapp.databinding.FragmentInteroperabilityConfigurationBinding
+import de.rki.coronawarnapp.ui.main.MainActivity
+import de.rki.coronawarnapp.ui.viewmodel.InteroperabilityViewModel
+
+class InteroperabilityConfigurationFragment : Fragment() {
+    companion object {
+        private val TAG: String? = InteroperabilityConfigurationFragment::class.simpleName
+    }
+
+    private val interoperabilityViewModel: InteroperabilityViewModel by activityViewModels()
+    private var _binding: FragmentInteroperabilityConfigurationBinding? = null
+    private val binding: FragmentInteroperabilityConfigurationBinding get() = _binding!!
```

Comment:
The `!!` seems to be error-prone to me. In fact, _binding can be null and this will crash. I'd prefer a single FragmentInteroperabilityConfigurationBinding using a lateinit var with public getter
`private lateinit var _binding: FragmentInteroperabilityConfigurationBinding`

Reply:
Will change this to the new behaviour @d4rken introduced. Will be in the next commit!